module-types/Cisco/N7K-AC-6KW.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Cisco
+model: N7K-AC-6.0KW
+part_number: N7K-AC-6.0KW
+comments: '[Cisco Nexus 7000 Series AC Power Supply Modules Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/Data_Sheet_C78-437761.html)'
+power-ports:
+  - name: Input 1
```

Comment:
This will throw a duplicate errror.

I prefer: PSU{module} for most PSU's, for this, perhaps PSU{module}-1

Reply:
I've made the change to prepend PSU{module} in front of Input 1